### ISSUE TITLE
Persist in-progress votes in voice kick command

### ISSUE THREAD

**Issue by ScoreUnder** [ScoreUnder/canti-bot#9]
The voice kick command does not persist in-progress votes. They should be persisted to the database so that on restart, the bot can do one of the following:

- Restore them to full functionality (allowing people to vote again)
    - This would still leave a period of non-functionality while the bot is restarting
- OR, close all votes
    - Closing it earlier (on shutdown) would delay shutdown waiting on API, which is something I want to avoid as much as possible

Relates to #2

**Comment by ScoreUnder** [comment]
This is probably too complex for too little gain, so for now wontfix
